install-dev/theme/views/content.php

```diff
@@ -0,0 +1,124 @@
+<?php
+  /**
+   * Copyright since 2007 PrestaShop SA and Contributors
+   * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+   *
+   * NOTICE OF LICENSE
+   *
+   * This source file is subject to the Open Software License (OSL 3.0)
+   * that is bundled with this package in the file LICENSE.md.
+   * It is also available through the world-wide-web at this URL:
+   * https://opensource.org/licenses/OSL-3.0
+   * If you did not receive a copy of the license and are unable to
+   * obtain it through the world-wide-web, please send an email
+   * to license@prestashop.com so we can send you a copy immediately.
+   *
+   * DISCLAIMER
+   *
+   * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+   * versions in the future. If you wish to customize PrestaShop for your
+   * needs please refer to https://devdocs.prestashop.com/ for more information.
+   *
+   * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+   * @copyright Since 2007 PrestaShop SA and Contributors
+   * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+   */
+?>
+
+<div id="infosShopBlock">
+  <h2><?php echo $this->translator->trans('Configure content', [], 'Install'); ?></h2>
+
+  <div class="field clearfix">
+    <label class="aligned"><?php echo $this->translator->trans('Select theme', [], 'Install'); ?></label>
```

Comment:
I think this is supposed to look like this with the word "Installation" at each step. For more consistency. 

![image](https://user-images.githubusercontent.com/70583503/148385759-3036a746-9b87-4886-bd9d-80b26068c208.png)

So the label would be "Installation of theme" and the help text: "Select the theme to install."